src/SIM/SIMoptions.C

```diff
@@ -141,6 +142,11 @@ bool SIMoptions::parseOutputTag (const TiXmlElement* elem)
     utl::getAttribute(elem,"dt",dtSave);
   }
 
+  else if (!strcasecmp(elem->Value(),"restartstride")) {
+    const char* value = utl::getValue(elem,"restartstride");
+    if (value) restartInc = atoi(value);
+  }
+
```

Comment:
See other places where I do this kind of parsing in a more compact way:
```
else if ((value = utl::getValue(elem,"stride"))) { 
  saveInc = atoi(value); 
  utl::getAttribute(elem,"dt",dtSave); 
}
else if ((value = utl::getValue(elem,"restartstride"))) 
  restartInc = atoi(value);
 ```
Then no need to check each keyword twice. Of course, need to define `const char* value` outside the loop first.